src/View/Widgets/BreadcrumbsEntry.vala

```diff
@@ -464,22 +464,19 @@ namespace Marlin.View.Chrome {
             foreach (AppInfo app_info in app_info_list) {
                 if (app_info != null && app_info.get_executable () != Environment.get_application_name ()) {
                     at_least_one = true;
-                    var menu_item = new Gtk.ImageMenuItem.with_label (app_info.get_name ());
+                     var item_grid = new Gtk.Grid ();
+                    item_grid.add (new Gtk.Image.from_gicon (app_info.get_icon (), Gtk.IconSize.MENU));
```

Comment:
I have the feeling that you'll need to also specify `pixel_size` to not allow the image to be bigger than Gtk.IconSize.MENU (happened in the past with Firefox and LibreOffice, to verify)